templates/local_settings.py.j2

```diff
@@ -54,3 +70,13 @@ CATALOGUE = {
 
 MEDIA_ROOT = "/var/www/{{ app_name }}/uploaded"
 STATIC_ROOT = "/var/www/{{ app_name }}/static"
+
+ALLOWED_HOSTS = ['localhost', '{{ server_name }}']
```

Comment:
Could you clarify your concern?  `local_settings.py` is needed until we fully move to environmental variables.  We shouldn't be patching settings.py in the deployed git repo, because it's under version control.  local_settings.py isn't under version control, since it should not be committed on the GitHub repo.